frontend/src/lib/constants.tsx

```diff
@@ -116,7 +116,6 @@ export const FEATURE_FLAGS = {
     KAFKA_INSPECTOR: 'kafka-inspector', // owner: @yakkomajuri
     INSIGHT_EDITOR_PANELS: '8929-insight-editor-panels', // owner: @mariusandra
     FRONTEND_APPS: '9618-frontend-apps', // owner: @mariusandra
-    ONBOARDING_1_5: 'onboarding-1_5', // owner: @liyiy
```

Comment:
this relies on @kappa90's remove onboarding PR?

Reply:
Ah there might be some nasty stash stuff going on here. I'll clean it up